ui/packages/shared/profile/src/ProfileView/ProfileViewContext.tsx

```diff
@@ -17,13 +17,11 @@ import {ProfileSource} from '../ProfileSource';
 
 interface Props {
   profileSource?: ProfileSource;
-  sampleUnit: string;
```

Comment:
Any specific reason to not replace this field with `profileType`? 

The reason we introduced this context is to avoid the unnecessary prop drilling for each one of the common data points. `profileType` sounds to fit that use case if I'm not mistaken.

Reply:
We're passing `profileSource`, which already containers the `profileType` via `profileSource?.ProfileType()`.
If this wasn't the case, I'd agree that we should pass it instead.

Reply:
Ah, ok didn't realize that.